soh/soh/Enhancements/cosmetics/CosmeticsEditor.cpp

```diff
@@ -2045,8 +2045,8 @@ void CosmeticsEditorWindow::DrawElement() {
                              UIWidgets::ComboboxOptions()
                                  .DefaultIndex(COLORSCHEME_N64)
                                  .Color(THEME_COLOR)
-                                 .LabelPosition(UIWidgets::LabelPosition::Near)
-                                 .ComponentAlignment(UIWidgets::ComponentAlignment::Right));
+                                 .LabelPosition(UIWidgets::LabelPositions::Near)
+                                 .ComponentAlignment(UIWidgets::ComponentAlignments::Right));
```

Comment:
What was the warning here exactly? Not opposed to the rename just curious what warning it was causing before

Reply:
type name & method name conflicted, `changes-meaning` is warning category

Reply:
OK, I thought that might've been it. This also changes the enums to line up with Colors in terms of being plural so I think that's fine.